introduction.md

```diff
@@ -0,0 +1,2 @@
+Hi, my name is {state your name}. I would like to join this dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.